sdk/azidentity/device_code_credential.go

```diff
@@ -15,27 +16,47 @@ const (
 	deviceCodeGrantType = "urn:ietf:params:oauth:grant-type:device_code"
 )
 
+// DeviceCodeCredentialOptions provide options that can configure DeviceCodeCredential instead of using the default values.
+type DeviceCodeCredentialOptions struct {
+	TenantID   string                  // Gets the Azure Active Directory tenant (directory) ID of the service principal
```

Comment:
Doc comments go above field declarations.

Reply:
I'll add this change now, but I will have to create a separate PR to do this for all of the other credentials as well.

Reply:
Done